<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KThemeScheduler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icons/heart.png</normaloff>:/resources/icons/heart.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="mainStack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="radioBtns">
          <item>
           <widget class="QRadioButton" name="scheduleRadioBtn">
            <property name="text">
             <string>Schedule</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sunRadioBtn">
            <property name="text">
             <string>Sunset/Sunrise</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <spacer name="bottomVerSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>164</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="topVerSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>165</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="lightTime">
          <item>
           <widget class="QLabel" name="lightTimeLabel">
            <property name="text">
             <string>Light:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEditLight"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="lightDarkBtns">
          <item>
           <widget class="QPushButton" name="lightBtn">
            <property name="text">
             <string>Light</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="darkBtn">
            <property name="text">
             <string>Dark</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="topSettings">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="prefsBtn">
            <property name="text">
             <string>Preferences</string>
            </property>
            <property name="icon">
             <iconset theme="preferences-other-symbolic"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="darkTime">
          <item>
           <widget class="QLabel" name="darkTimeLabel">
            <property name="text">
             <string>Dark:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEditDark"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="autoCheckBox">
          <property name="text">
           <string>Automatic</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="startupCheckBox">
            <property name="text">
             <string>Start KThemeScheduler on system startup.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="backBtn">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-84</y>
             <width>398</width>
             <height>564</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0" colspan="2">
             <layout class="QVBoxLayout" name="plasmaStyle">
              <item>
               <widget class="QCheckBox" name="styleCheckBox">
                <property name="text">
                 <string>Plasma Style</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="lightStyle">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Light:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="darkStyle">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Dark:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QComboBox" name="lightDropStyle">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="darkDropStyle">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QVBoxLayout" name="colorScheme">
              <item>
               <widget class="QCheckBox" name="colorCheckBox">
                <property name="text">
                 <string>Color Scheme</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="lightColor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Light:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="darkColor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Dark:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QComboBox" name="lightDropColor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="darkDropColor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QVBoxLayout" name="iconTheme">
              <item>
               <widget class="QCheckBox" name="iconCheckBox">
                <property name="text">
                 <string>Icon Theme</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="lightIcon">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Light:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="darkIcon">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Dark:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QComboBox" name="lightDropIcon">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="darkDropIcon">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="3" column="0" colspan="2">
             <layout class="QVBoxLayout" name="cursorTheme">
              <item>
               <widget class="QCheckBox" name="cursorCheckBox">
                <property name="text">
                 <string>Cursor Theme</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="lightCursor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Light:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="darkCursor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Dark:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QComboBox" name="lightDropCursor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="darkDropCursor">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="4" column="0" colspan="2">
             <layout class="QVBoxLayout" name="gtkTheme">
              <item>
               <widget class="QCheckBox" name="gtkCheckBox">
                <property name="text">
                 <string>GTK Theme</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="lightGtk">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Light:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="darkGtk">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Dark:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QComboBox" name="lightDropGtk">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="darkDropGtk">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="5" column="0" colspan="2">
             <layout class="QVBoxLayout" name="wallpaper">
              <item>
               <widget class="QCheckBox" name="wallCheckBox">
                <property name="text">
                 <string>Wallpaper</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="lightWall">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Light:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="darkWall">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Dark:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QComboBox" name="lightDropWall">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="darkDropWall">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="applyBtn">
            <property name="text">
             <string>Apply</string>
            </property>
            <property name="icon">
             <iconset theme="dialog-apply"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelBtn">
            <property name="text">
             <string>Cancel</string>
            </property>
            <property name="icon">
             <iconset theme="dialog-cancel"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPrefs"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionHide"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="window-close-symbolic"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about-symbolic"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>?</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="icon">
    <iconset theme="window-minimize-symbolic"/>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionPrefs">
   <property name="icon">
    <iconset theme="preferences-other-symbolic"/>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
